### PR TITLE
Update direct-routing-sbc-multiple-tenants.md

### PR DESCRIPTION
**Pull request by lindspea**
https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/issues/6247
Added note.

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/direct-routing-sbc-multiple-tenants.md [hunk 1]
```diff
@@ -191,6 +191,9 @@ For more information about admin roles and how to assign a role in Microsoft 365
 11. Ensure that the status is **Setup complete**. 
     
     ![Screenshot of page showing status of Setup complete](media/direct-routing-12-sbc-setup-complete.png)
+    
+> [!NOTE]
+> The Base URL and the subdomain for the individual client has to be on the same tenant for you to add a 'direct route' trunk in.
```
Comment (JohanFreelancer9) [inline Teams/direct-routing-sbc-multiple-tenants.md:196]:
```suggestion
> The base URL and the subdomain for the individual client has to be on the same tenant for you to add a 'direct route' trunk.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

File: Teams/direct-routing-sbc-multiple-tenants.md [hunk 2]
```diff
@@ -191,6 +191,9 @@ For more information about admin roles and how to assign a role in Microsoft 365
 11. Ensure that the status is **Setup complete**. 
     
     ![Screenshot of page showing status of Setup complete](media/direct-routing-12-sbc-setup-complete.png)
+    
+> [!NOTE]
+> The base URL and the subdomain for the individual client has to be on the same tenant for you to add a _direct route_ trunk.
```
Comment (JohanFreelancer9) [inline Teams/direct-routing-sbc-multiple-tenants.md:196]:
```suggestion
> The base URL and the subdomain for the individual client have to be on the same tenant for you to add a _direct route_ trunk.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Apologies @lindspea - some more, please.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@CarolynRowe PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**